docs/_extensions/lazy_autodoc.py

```diff
@@ -153,7 +155,7 @@ def process_roles(file: Path) -> List[Tuple[str, str]]:
     # add members to their parents
 
     api_preamble = """
-    """
+"""
```

Comment:
intended change?

Reply:
Indeed! This fixes incorrect rendering in the API documentation (see the heading `## Classes` in https://acts.readthedocs.io/en/latest/api/api.html)